cumulusci/core/tests/test_github.py

```diff
@@ -31,8 +40,9 @@ def isclosed(self):
         return True
 
 
-class TestGithub(unittest.TestCase):
-    def tearDown(self):
+class TestGithub:
+    @classmethod
+    def teardown(cls):
```

Comment:
Can you confirm that this is being called after each test? I think it might need to be defined like this:

    def teardown_method(self, method):